docs/design/autoware-architecture/localization/index.md

```diff
@@ -1,5 +1,294 @@
+| base_link | Reference pose of the ego-vehicle (projection of the rear-axle center onto the ground surface) |
+|  sensor   | Reference pose of each sensor                                                                  |
+
+Developers can optionally add other frames such as odom or base_footprint while keeping the tf structure above.
+
+### The localization module's ideal functionality
+
+- The localization module should provide pose, velocity, and acceleration for control, planning, and perception
+- Latency and stagger should be sufficiently small or adjustable so that the estimated values can be used for control within the ODD(Operational Design Domain)
+- The localization module should produce the pose on a fixed coordinate frame
+- Sensors should be independent of each other and keep the replaceability
+- The localization module should provide the status indicating if the autonomous vehicle can operate or not with the self-contained function or map information.
+- Tools or manuals should describe how to set proper parameters for the localization module <!-- TODO  原文にある「適切な位置推定」とはなんだ！！！ -->
+- Valid calibration parameters should be provided to align different frame or pose coordinates and sensor timestamps.
+
+### KPI
+
+We jointly consider these metrics if the system keeps sufficient pose estimation performance for safe operation.
+
+- Safety
+  - The percentage of distance which the pose estimation satisfies the required accuracy per the overall distances defined in ODD
+  - The anomaly detection rate of the case that the localization module cannot estimate the pose within ODD
+  - The accuracy of out-of-ODD detection
+- Computational load
+- Latency
+
+## 5. Interface and Data Structure
+
+## 6. Concerns, Assumptions, and Limitations
+
+### Prerequisites of sensors and inputs
+
+#### Sensor prerequisites
+
+- Input data is not defective. Internal sensor observation such as IMU continuously keeps the proper frequency
+- Input data has correct and exact time stamps. Estimated poses can be inaccurate or unstable if the timestamps are not exact
+- Sensors are correctly mounted on the exact positions and accessible from TF. If the sensor positions are inaccurate, estimation results may be incorrect or unstable. A sensor calibration framework is required to properly obtain the sensor positions.
+
+#### Map prerequisites
+
+- Pose estimation might be unstable if there is no sufficient information on the map. A testing framework is demanded to check if the map has adequate information for pose estimation.
+- Pose estimation might be unstable if the actual environment has different objects from the map. Maps need updates according to new objects and seasonal changes.
+- If multiple maps with different coordinate systems are used, the misalignment between them can affect the localization performance. Maps have to be aligned to a uniform coordinate, or we require an alignment framework.
```

Comment:
The phrasing of the map prequisites is different to that for the sensor prerequisites and talks about the problems first before the prequisite, so I have reordered the sentences accordingly.

```suggestion
- Sufficient information is contained within the map.
  - Pose estimation might be unstable if there is insufficient information in the map. 
  - A testing framework is necessary to check if the map has adequate information for pose estimation.
- Map does not differ greatly from the actual environment.
  - Pose estimation might be unstable if the actual environment has different objects from the map. 
  - Maps need updates according to new objects and seasonal changes.
- Maps must be aligned to a uniform coordinate, or an alignment framework is in place. 
  - If multiple maps with different coordinate systems are used, the misalignment between them can affect the localization performance.
```